src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
+	
+	/** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
+    static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
+
+	/**
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.